licenses/git_utils.py

```diff
@@ -1,4 +1,23 @@
+import os
+
 import git
+from django.conf import settings
+
+
+def setup_to_call_git():
+    """
+    Call this to set the environment before starting to use git.
+    Safe to call any number of times.
+    """
+    # Use custom ssh command to use the deploy key when pushing
+    if "GIT_SSH" not in os.environ:
+        os.environ["GIT_SSH"] = os.path.join(settings.ROOT_DIR, "ssh_wrapper.sh")
+    if "TRANSLATION_REPOSITORY_DEPLOY_KEY" not in os.environ:
+        os.environ[
+            "TRANSLATION_REPOSITORY_DEPLOY_KEY"
+        ] = settings.TRANSLATION_REPOSITORY_DEPLOY_KEY
+    if "PROJECT_ROOT" not in os.environ:
+        os.environ["PROJECT_ROOT"] = settings.ROOT_DIR
```

Comment:
Can't this be defined in settings? We can supply a default to the `.get` method as the second parameter.